tst/EnergyPlus/unit/IdfParser.unit.cc

```diff
@@ -83,7 +83,7 @@ TEST_F(IdfParserFixture, decode_success)
 {
     bool success = true;
     std::string const test_object(delimited_string({
-        "Version,8.3;",
+        "Version," + DataStringGlobals::MatchVersion + ";",
```

Comment:
I updated all but 2-3 places where it wasn't as convenient so we don't need to update these again. As you can see, many were very far out of date, so I'm not sure what value having these hard coded into our tests brings.